From 1a0aa89304eba1b10a0d122501af7f2b96b36fda Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.org>
Date: Sat, 17 Aug 2019 19:47:30 +0100
Subject: [PATCH 770/773] overlays: sc16ic752-i2c: Fix xtal parameter

The xtal parameter is targetting the wrong node - fix it.

See: https://github.com/raspberrypi/linux/issues/3156

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 arch/arm/boot/dts/overlays/sc16is752-i2c-overlay.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/overlays/sc16is752-i2c-overlay.dts
+++ b/arch/arm/boot/dts/overlays/sc16is752-i2c-overlay.dts
@@ -35,6 +35,6 @@
 	__overrides__ {
 		int_pin = <&sc16is752>,"interrupts:0";
 		addr = <&sc16is752>,"reg:0",<&sc16is752_clk>,"name";
-		xtal = <&sc16is752>,"clock-frequency:0";
+		xtal = <&sc16is752_clk>,"clock-frequency:0";
 	};
 };
